auto_round/mllm/autoround_mllm.py

```diff
@@ -161,8 +162,17 @@ def __init__(
             enable_norm_bias_tuning: bool = False,
             truncation: bool = None,
             enable_torch_compile: bool = False,
+            model_kwargs: dict = None,
             **kwargs,
     ):
+        if isinstance(model, str):
+            torch_dtype = "auto"
+            if device is not None and "hpu" in device:
+                torch_dtype = torch.bfloat16
+            model, processor, tokenizer, image_processor = mllm_load_model(
+                model, torch_dtype=torch_dtype, **model_kwargs)
+
+        assert tokenizer is not None, "tokenizer should not be None."
```

Comment:
This conflicts with the default value. Besides, the assertion might cause issues for the INC part, as they pass in tokenized text directly.